consensus/dummy/dynamic_fees.go

```diff
@@ -14,6 +15,12 @@ import (
 	"github.com/ethereum/go-ethereum/common/math"
 )
 
+var (
+	MaxUint256Plus1                     = new(big.Int).Lsh(common.Big1, 256)
+	MaxUint256                          = new(big.Int).Sub(MaxUint256Plus1, common.Big1)
```

Comment:
Unless absolutely necessary, these shouldn't be exported. They're effectively constants but are now global (beyond even the package) mutable variables.